src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -95,10 +112,15 @@ public void satisfiesEqualsContract() throws IOException {
                 )
             )
         );
-        MatcherAssert.assertThat(
-            new RqMultipart.Base(req),
-            Matchers.equalTo(new RqMultipart.Base(req))
-        );
+        final RqMultipart.Base one = new RqMultipart.Base(req);
+        final RqMultipart.Base two = new RqMultipart.Base(req);
+        try {
+            MatcherAssert.assertThat(one, Matchers.equalTo(two));
+        } finally {
```

Comment:
@raphaelln same queston here - do we really need to use finally here? This just ensures closing being done on assertion failure, but on assertion failure test is failed anyway so we don't care that much it think.

The only exception would be if after assertion failure some files were stored on disk permanently.